packages/browser/src/core/queue/event-queue.ts

```diff
@@ -232,9 +232,16 @@ export class EventQueue extends Emitter {
         return true
       }
 
+      const alternativeNameMatch =
+        p.alternativeNames &&
+        p.alternativeNames.filter((name) =>
```

Comment:
Optional changing nitpick! p.alternativeNames?.filter would be a bit nicer IMO

Reply:
I typically agree - though the size savings are not insignificant due to the transpilations that occur (assuming variables/field names get minified). See [this](https://www.typescriptlang.org/play?target=1#code/JYOwLgpgTgZghgYwgAgGIHt3IN4ChnIBGA-AFzIAUAlMgLwB8yAbusACa4C+uuC6IAZzDI45DFlo4i5anUbZOybrjgA6Equor1yAGS6R66kA) as an example. (Just a tradeoff - if it makes the code more readable then go with the optional chaining - I was just surprised how much larger the transpiled code was)

Reply:
interesting...